paywall/src/data-iframe/blockchainHandler/web3ServiceHub.js

```diff
@@ -1,5 +1,13 @@
-import { getTransactions, getAccount } from '../cacheHandler'
+import { getAccount } from '../cacheHandler'
 
+let transactionCache = {}
+
+/**
+ * for unit testing purposes
+ */
+export function __clearCache() {
```

Comment:
Generally we should not add method solely for testing purposes. Another approach here would be possibly passing the transactionCache to web3ServiceHub

Reply:
great idea!